pkg/quota/resources.go

```diff
@@ -251,7 +251,7 @@ func CalculateUsage(namespaceName string, scopes []api.ResourceQuotaScope, hardL
 	for _, evaluator := range evaluators {
 		potentialResources = append(potentialResources, evaluator.MatchingResources(hardResources)...)
 	}
-	// NOTE: the intersection just removes duplicates since the evaluator match intersects wtih hard
+	// NOTE: the intersection just removes duplicates since the evaluator match intersects with hard
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464. Closing.

/close